nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/global_time_expired_condition.hpp

```diff
@@ -55,13 +56,15 @@ class GlobalTimeExpiredCondition : public BT::ConditionNode
   static BT::PortsList providedPorts()
   {
     return {
-      BT::InputPort<double>("seconds", 1.0, "Seconds")
+      BT::InputPort<double>("seconds", 1.0, "Seconds"),
+      BT::InputPort<nav_msgs::msg::Path>("path")
```

Comment:
I think this node needs a new name if its now specific to the path. Brainstorm some ideas?